android/app/src/main/java/au/com/robin/sms/SmsSend.kt

```diff
@@ -0,0 +1,47 @@
+package au.com.robin.sms
+
+import android.Manifest
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.telephony.SmsManager
+import android.telephony.SubscriptionManager
+import android.util.Log
+import androidx.annotation.RequiresPermission
+
+class SmsSend {
+
+    companion object {
```

Comment:
What's a companion object?

Reply:
> What's a companion object?

It lets us use the instance of this without instantiating it everytime we want to reference it in different class. It is a singleton object.

Reply:
Thanks for the explanation. Place a quick comment about it. I have opinions about it, but I'll leave it for next time.

Reply:
It basically like a standalone function right? Can you create functions (without needing a class) in Kotlin?

Reply:
> It basically like a standalone function right? Can you create functions (without needing a class) in Kotlin?

Yes, we actually can. So, maybe just make a standalone function?

Reply:
I leave that up to you.